openapi/components/schemas/CheckoutForm/StorefrontCheckoutForm.yaml

```
@@ -6,3 +6,11 @@ allOf:
         description: The session's token used for authentication
         type: string
         readOnly: true
+      _links:
+        type: array
+        description: The links related to resource
```

```suggestion
        description: The links related to resource.
```

I think we need a period there (and other places)

https://rebilly.slack.com/archives/C7U0AHYJ0/p1601565453042200

---

```suggestion
        description: The links related to resource.
```